cfme/fixtures/v2v_fixtures.py

```diff
@@ -523,7 +523,10 @@ def mapping_data_dual_vm_obj_dual_datastore(request, appliance, source_provider,
 
 @pytest.fixture(scope="function")
 def mapping_data_vm_obj_dual_nics(request, appliance, source_provider, provider,
-                                  source_type, dest_type, template_type):
+                                  template_type=Templates.DUAL_NETWORK_TEMPLATE):
```

Comment:
I don't think this is good practice i.e. providing default value to fixture.
pytest doesn't consider arguments with default values as targets for fixtures/parameterization.